client/src/containers/HomePage.tsx

```diff
@@ -136,11 +148,11 @@ class HomePage extends Component<HomePageProps, State> {
         <Redirect
           push
           to={{
-            pathname: createRouteForAddressPage(
-              this.state.searchAddress.boro,
-              this.state.searchAddress.streetname,
-              this.state.searchAddress.housenumber
-            ),
+            pathname: createRouteForAddressPage({
+              boro: this.state.searchAddress.boro,
+              streetname: this.state.searchAddress.streetname,
+              housenumber: this.state.searchAddress.housenumber,
+            }),
```

Comment:
Similarly, this would be a lot easier to read as `createRouteForAddressPage(this.state.searchAddress)`!